bootstrap/kubeadm/api/v1alpha3/kubeadmbootstrapconfig_types.go

```diff
@@ -49,6 +49,14 @@ type KubeadmConfigSpec struct {
 	// +optional
 	Files []File `json:"files,omitempty"`
 
+	// FileSources specifies external sources to populate files on the
+	// target host system. Each source may produce multiple files. Data
+	// reused frequently or sensitive data not appropriate for storage
+	// in plain text may be stored in external sources and resolved at
+	// runtime.
+	// +optional
+	FileSources []FileSource `json:"fileSources,omitempty"`
```

Comment:
Should this be `FilesFrom`?

Reply:
That's what was discussed in the other PR but I felt the alignment with volumes with configmap and secret source made sense, no reason to diverge